test/unit/offset.js

```diff
@@ -462,9 +462,9 @@ testIframe("offset/body", "body", function( $ ) {
 test("chaining", function() {
 	expect(3);
 	var coords = { "top":  1, "left":  1 };
-	equal( jQuery("#absolute-1").offset(coords).selector, "#absolute-1", "offset(coords) returns jQuery object" );
-	equal( jQuery("#non-existent").offset(coords).selector, "#non-existent", "offset(coords) with empty jQuery set returns jQuery object" );
-	equal( jQuery("#absolute-1").offset(undefined).selector, "#absolute-1", "offset(undefined) returns jQuery object (#5571)" );
+	equal( jQuery("#absolute-1").offset(coords).jquery, jQuery.fn.jquery, "offset(coords) returns jQuery object" );
```

Comment:
> This property indeed serve for two complitly different things, but i don't think that's good, i think these should be exposed explicitly then otherwise.

The purposes aren't completely different; they're analogous to `nodeType` checks in that both properties are considered to have a universal meaning regardless of where they appear. `jquery` specifies the version of jQuery used to construct a collection, which—since it must be `undefined` for objects that are _not_ jQuery collections—can be cast as a boolean for type checking, just like casting `nodeType` to boolean checks for DOM nodes. That said, though, there might well have been a better choice for the name, but it's definitely too late to change.